app-backup/luckybackup/luckybackup-0.4.9.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="A powerful, fast and reliable backup & sync tool"
+DESCRIPTION="An application that backs-up and/or synchronizes any directories with the power \
+of rsync. It is simple to use, fast (transfers over only changes made and not all data), safe\
+ (keeps your data safe by checking all declared directories before proceeding in any data \
+manipulation), reliable and fully customizable."
+COPYRIGHT="2008-2017 Loukas Avgeriou"
+LICENSE="GNU GPL v3"
+REVISION="1"
+HOMEPAGE="http://luckybackup.sourceforge.net"
+SOURCE_URI="https://downloads.sourceforge.net/luckybackup/luckybackup-$portVersion.tar.gz"
+CHECKSUM_SHA256="c161e98a11de5ad0a32a8e060e3fd45695a93ea3adf0581764a6b5d32e8da23b"
+ADDITIONAL_FILES="luckybackup.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
```

Comment:
Suggestion:
```diff
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
 SECONDARY_ARCHITECTURES="x86"
+fi
```

Reply:
The purpose of the above suggestion is to be friendly to users of x86_64 hybrid, even if builds of that platform are not yet available.
What "friendly" means is that the repository cache won't be cluttered with recipes caches which are not needed: Indeed, on x86_64 hybrid, we don't need `luckybackup_x86` because we will have `luckybackup`.